spec/factories/institution_factory.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :institution do
+    name { Faker::FunnyName.name }
```

Comment:
Up to you, but IMO, faker in tests brings very little value, and it makes it way harder to debug failing tests, as every time a test fails, you'll get a different error message. I prefer to either hardcode factory's values or use a sequence

Reply:
ok but in some tests I need to generate lets say 20 institutions. How do i then use it? Hardcoding 20 names seems excessive. Can it be left to faker as default but overridden in some tests where it is useful? I think that leaving this as a default is fine.

Reply:
You can use `sequence` if you need to have unique value in the database
```suggestion
  factory :institution do
    sequence(:name) { |n| "LO #{n}" }
```

Reply:
BTW, if you want me to add some additional clarification to an old comment, please do not resolve it. I'm usually not checking resolved comments, and I'll probably miss those followups